### DISCUSSION TITLE
Problem With Running the Example Code

### DISCUSSION THREAD

**Discussion by Ahmad-197** [cubert-hyperspectral/cuvis.sdk#60] [Q&A]
Hi,

I tried to run the  EX05_recordSingleImages.py file, the code did not give any error but it also failed to capture any photos as well. It kept on giving two warning messages. The terminal output is given below:

<img width="479" alt="image" src="https://github.com/cubert-hyperspectral/cuvis.sdk/assets/86935871/74bc53c7-5876-4a38-a923-0b88b2114b85">

Can anyone tell me what might be the issue and what should I do resolve it?

**Comment by birkholz-cubert** [comment]
Hey,

This seems to be an error not related to the sdk, but rather to your specific camera setup. For this you should open an issue at our service portal http://support.cubert-hyperspectral.com/.
